### PR TITLE
chore: add anon_urls.txt file for vrt

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-9700

### CODE REVIEW

**Review by cafuego — DISMISSED** [review]

File: tests/vrt/anon_urls.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+https://reliefweb.int/organizations
+https://reliefweb.int/topics
+https://reliefweb.int/blog
+https://reliefweb.int/report/occupied-palestinian-territory/delayed-denied-and-deprived-collective-punishment-palestinian-patients-gaza-context-israels-15-year-blockade
+https://reliefweb.int/map/myanmar/myanmar-place-origin-townships-displaced-population-camp-profiling-round-12-kachin-and-shan-state-north-may-2022
+https://reliefweb.int/job/3856494/ukraine-crisis-response-medical-coordinator-mf
+https://reliefweb.int/training/3850078/organised-crime-and-security-policy-understanding-threat-design-better-responses
+https://reliefweb.int/blogpost/onwards-and-upwards-reliefweb-turns-25-record-numbers
+https://reliefweb.int/country/afg
+https://reliefweb.int//disaster/fl-2021-000084-tcd
```
Comment (orakili) [inline tests/vrt/anon_urls.txt:16]:
```suggestion
https://reliefweb.int/disaster/fl-2021-000084-tcd
```

File: tests/vrt/anon_urls.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+https://reliefweb.int/organization/ocha
+https://reliefweb.int/topics/covid-19-global
+https://reliefweb.int/updates?advanced-search=%28C75%29_%28S1503.S1275%29_%28T4595%29&search=ebola
+https://reliefweb.int/announcement/my-hero-you-storybook-children-covid-19
```
Comment (orakili) [inline tests/vrt/anon_urls.txt:20]: This shows the 404 page for anonymous users. Is that what you want?

**Review by orakili — CHANGES_REQUESTED** [review]
I would add:

- https://reliefweb.int/help
- https://reliefweb.int/search/results?search=humanitarian
- https://reliefweb.int/admin/config (403 for anonymous users)
- https://reliefweb.int/unexisting-page (404)

Note that most of the URLs in that list have content that changes frequently. Not sure how important that is for VRT.

**Comment by cafuego** [comment]
auto-VRT for reliefweb is going to need a bit of love; the database would need to be the lite one by default, or the whole process ends up taking half a day. That also means the URLs to be tested need to be for content in the lite database.

File: tests/vrt/anon_urls.txt (hunk 1 above)
Comment (lazysoundsystem) [reply tests/vrt/anon_urls.txt]: Fixed (though it's re-directs to the right url automatically, which is how we hadn't picked it up before)

File: tests/vrt/anon_urls.txt (hunk 2 above)
Comment (lazysoundsystem) [reply tests/vrt/anon_urls.txt:22]: It wasn't the intention, but I think it's okay to leave it.

**Comment by lazysoundsystem** [comment]
I tested all the urls against the lite database and they're fine - except for individual jobs and trainings, which keep going out of date. There are a couple of jobs with closing dates well into the future - https://reliefweb.int/jobs?advanced-search=%28DC20290101-%29 but at least one of those is likely to be a typo.

**Review by cafuego — DISMISSED** [review]

**Review by cafuego — APPROVED** [review]

**Review by orakili — APPROVED** [review]

**Pull request merged**